<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateBurrow</class>
 <widget class="QWidget" name="CreateBurrow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="burrowRandom">
   <property name="geometry">
    <rect>
     <x>248</x>
     <y>61</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="burrowCancel">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>111</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="burrowLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLabel" name="burrowNamelabel">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>11</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Burrow:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="burrowCreate">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>111</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; padding: 5px; background: rgb(30, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="burrowClose">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0; color: white; background: rgb(30, 0, 0)</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>burrowRandom</zorder>
  <zorder>burrowCancel</zorder>
  <zorder>burrowLineEdit</zorder>
  <zorder>burrowNamelabel</zorder>
  <zorder>burrowCreate</zorder>
  <zorder>burrowClose</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
